integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -30,3 +30,191 @@ Create chart name and version as used by the chart label.
 {{- define "alluxio.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
```

Comment:
Most duplicated blocks are moved into this templating field instead of staying in the YAML templates.